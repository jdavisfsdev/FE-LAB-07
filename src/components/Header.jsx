import React from 'react';
import { Link } from 'react-router-dom';

export default function Header() {
  return (
    <div>
      <Link to="/">
        <button>HOME</button>   
      </Link>   
    </div>
  );
}
