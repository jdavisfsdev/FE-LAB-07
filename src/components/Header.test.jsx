import React from 'react';
import { render } from '@testing-library/react';
import Header from './Header';
import { MemoryRouter } from 'react-router-dom';

describe('Header component snapshot', () => {
  it('render Header', () => {
    const { asFragment } = render(
      <MemoryRouter>
        <Header />
      </MemoryRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
