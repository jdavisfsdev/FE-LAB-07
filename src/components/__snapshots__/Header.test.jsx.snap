// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component snapshot render Header 1`] = `
<DocumentFragment>
  <div>
    <a
      href="/"
    >
      <button>
        HOME
      </button>
    </a>
  </div>
</DocumentFragment>
`;
